builtin/providers/aws/resource_aws_route53_delegation_set_test.go

```diff
@@ -0,0 +1,153 @@
+package aws
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/service/route53"
+)
+
+func TestAccRoute53DelegationSet(t *testing.T) {
```

Comment:
and you add a `_basic` suffix here? Otherwise we can't easily run `TestAccRoute53DelegationSet` in insolation